src/partition/repart.c

```diff
@@ -312,6 +312,8 @@ typedef struct Partition {
         MinimizeMode minimize;
         uint64_t verity_data_block_size;
         uint64_t verity_hash_block_size;
+        char *compression;
+        uint64_t compression_level;
```

Comment:
hmm, are you sure the compression level is always an integer? don't i remember some compression alg that knew weird levels such as "0", "1", …, "9", "9a", "9b", "9c" or so? or was i hallucinating?

maybe play dumb here, just pass this through as string?